### DISCUSSION TITLE
[吐槽]三元老在打零号空洞妮妮薇实在是太难打了

### DISCUSSION THREAD

**Discussion by LShanShi** [share121/inter-knot#2999] [其他]
我是狼哥比利双推人,狼0星比利2星配队再加上妮可2星和狡兔屋2星帮布,全员专武+50级满天赋满技能+满级攻略过的驱动盘打妮妮薇疯狂暴毙闪避按烂打了快十分钟

**Comment by nostalgiatan** [comment]
你重复了一个帖子，可以删去一个，谢谢。

**Comment by RaphaelHider** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >同提醒帖子重复了。零号空洞鸣徽很关键。能量、闪避、护盾类型鸣徽可以让你打得过，暴击、对应属性、决斗等鸣徽可以让你打得更快。
说一下我的几个战斗经验：
1.尽量贴身打，方便格挡，以及它的激光是向前攻击，放这一招的时候可以绕到身后输出。有黄光尽量格挡，如果碰上转圈技能离得够近可以连续格挡13下左右。
2.优先清理小怪和触手。
3.探索过程中注意空洞侵蚀，避免带着不利于打boss的debuff进入战斗，例如闪避冷却加长、满侵蚀的尼尼微特攻。
4.狼哥特殊技可以增加怪物失衡状态易伤，尽量让狼哥击破或连携到再用主C输出。
祝你探索顺利，在空洞内永不迷失~

**Comment by Nie7bai** [comment]
鸣徽的话主要去找[闪避]羽毛球（S）、和[护盾]战术口罩（S），这俩比较关键，如果可能的话把拿4护盾开一下战术背包（有盾的时候就是小名刀），输出鸣徽的话随便拿就行，慢慢刮就是了
祝你探索顺利，在空洞内永不迷失~<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >
